source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -3252,12 +3252,16 @@ int KX_GameObject::pyattr_set_lodManager(PyObjectPlus *self_v, const KX_PYATTRIB
 	return PY_SET_ATTR_SUCCESS;
 }
 
-PyObject *KX_GameObject::PyApplyForce(PyObject *args)
+PyObject *KX_GameObject::PyApplyForce(PyObject *args, PyObject *kwds)
 {
 	int local = 0;
 	PyObject *pyvect;
 
-	if (PyArg_ParseTuple(args, "O|i:applyForce", &pyvect, &local)) {
+    static const char *kwlist[] = {"force", "local", nullptr};
```

Comment:
As for `addDebugProperty` i'm not sure of the interest of making the second argument a keyword. If the doc say the opposite you can change the doc like:
`applyBlah(vector[, local])`